### ISSUE TITLE
Not returning right exit code of nightwatch

### ISSUE THREAD

**Issue by ondrejbartas** [tatsuyafw/gulp-nightwatch#30]
I had to add callback function into `lib/background.js`

`nightwatch.runner(argv, function(err) { if (!err) {process.exit(1)}});`

so nighwatch is now exiting correctly if there was failed jobs

**Comment by tatsuyafw** [comment]
Hi @ondrejbartas .

Thanks for you feedback. I'll try to fix this issue.

**Comment by tatsuyafw** [comment]
@ondrejbartas Sorry for late response..

I've fixed this issue, and released as `v0.2.7`.

**Comment by ondrejbartas** [comment]
@tatsuyafw thank yo

**Comment by peterbollen** [comment]
I think there is a problem with this merge.
My build keeps hanging when all tests are green.

```
nightwatch.runner(argv, function(success) {
    if (!success) {
      process.exit(1);
    }
  });
```

Using this instead, solves my issue.
```
nightwatch.runner(argv, function(success) {
    if (!success) {
      process.exit(1);
    } else {
      process.exit(0);
    }
  });
```

**Comment by tatsuyafw** [comment]
This issue is fixed by #34 .
